### PR TITLE
op: Implement 'color_write_mask,channel_work' in color_target_state.spec.ts

### PR DESCRIPTION
**Pull request by Gyuyoung**
This PR adds a new test to ensure that write masks works alone or with multiple channels.

Issue: #1835

<hr>

**Requirements for PR author:**

- [x] All missing test coverage is tracked with "TODO" or `.unimplemented()`.
- [x] New helpers are `/** documented */` and new helper files are found in `helper_index.txt`.
- [x] Test behaves as expected in a WebGPU implementation. (If not passing, explain above.)

**Requirements for [reviewer sign-off](https://github.com/gpuweb/cts/blob/main/docs/reviews.md):**

- [x] Tests are properly located in the test tree.
- [x] [Test descriptions](https://github.com/gpuweb/cts/blob/main/docs/intro/plans.md) allow a reader to "read only the test plans and evaluate coverage completeness", and accurately reflect the test code.
- [x] Tests provide complete coverage (including validation control cases). **Missing coverage MUST be covered by TODOs.**
- [x] Helpers and types promote readability and maintainability.

When landing this PR, be sure to make any necessary issue status updates.

### CODE REVIEW

File: src/webgpu/api/operation/rendering/color_target_state.spec.ts [hunk 1]
```diff
@@ -621,6 +622,104 @@ g.test('blend_constant,not_inherited')
...
+          break;
+        case GPUColorWrite.ALPHA:
+          a = 1;
+          break;
+      }
+    }
+
+    const testPipeline = t.createRenderPipelineForTest({
+      format,
+      blend: { color: blendComponent, alpha: blendComponent },
```
Comment (kainino0x) [inline src/webgpu/api/operation/rendering/color_target_state.spec.ts:675]: I don't think blending is actually needed for this test, and you could remove `blend` from the descriptor entirely.

File: src/webgpu/api/operation/rendering/color_target_state.spec.ts [hunk 2]
```diff
@@ -621,6 +622,104 @@ g.test('blend_constant,not_inherited')
...
+          },
+        ],
+      });
+      renderPass.setPipeline(testPipeline);
+      renderPass.setBindGroup(
+        0,
+        t.createBindGroupForTest(testPipeline.getBindGroupLayout(0), kBaseColorData)
+      );
+      // Draw [1,1,1,1] with `src * 1 + dst * 1`. So the
+      // result is `[1,1,1,1] * [1,1,1,1] + [0,0,0,0] * [1,1,1,1]` = [1,1,1,1].
```
Comment (kainino0x) [inline src/webgpu/api/operation/rendering/color_target_state.spec.ts:704]: (remove this comment if you disable blending)

**Review by kainino0x — APPROVED** [review]

File: src/webgpu/api/operation/rendering/color_target_state.spec.ts (hunk 1 above)
Comment (Gyuyoung) [reply src/webgpu/api/operation/rendering/color_target_state.spec.ts:675]: Ah, I see. Removed!

File: src/webgpu/api/operation/rendering/color_target_state.spec.ts (hunk 2 above)
Comment (Gyuyoung) [reply src/webgpu/api/operation/rendering/color_target_state.spec.ts]: Done.

**Pull request merged**